src/index.html

```diff
@@ -22,6 +22,408 @@
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+
+    <main>
+      <section class="catalog">
+        <article
+          class="card"
+          data-qa="card"
+        >
+          <img
+            src="./images/imac.jpeg"
+            alt="card_image"
+            class="card__image"
+          />
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__rating">
+            <ul class="stars stars--4">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+
+            <p class="card__reviews">Reviews: 5</p>
+          </div>
+
+          <p class="card__price">
+            <span class="card__price-title">Price:</span>
+            <span class="card__price-value">$2,199</span>
+          </p>
+
+          <a
+            href="#"
+            class="card__button"
+            data-qa="card-hover"
+          >
+            BUY
+          </a>
+        </article>
+        <article class="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="card_image"
+            class="card__image"
+          />
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__rating">
+            <ul class="stars stars--4">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+
+            <p class="card__reviews">Reviews: 5</p>
+          </div>
+
+          <p class="card__price">
+            <span class="card__price-title">Price:</span>
+            <span class="card__price-value">$2,199</span>
+          </p>
+
+          <a
+            href="#"
+            class="card__button"
+            data-qa="card-hover"
+          >
+            BUY
+          </a>
+        </article>
+        <article class="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="card_image"
```

Comment:
``alt`` attribute should describe the image content

![image](https://github.com/mate-academy/layout_catalog/assets/54860641/d177874d-3f35-4507-afb2-72c5b083acd3)

Example:
```suggestion
            alt="APPLE A1419 iMac 27" Retina 5K Monoblock"
```